Source/DafnyStandardLibraries/src/DafnyStdLibs_FileIOInternalExterns.py

```diff
@@ -0,0 +1,37 @@
+# Module: DafnyStdLibs_FileIOInternalExterns
+
+import traceback
+import os
+import os.path
+import _dafny
+
+class default__:
+  @staticmethod
+  def INTERNAL__ReadBytesFromFile(path):
+    path_str = path.VerbatimString(False)
+    try:
+      with open(path_str, mode="rb") as file:
+          contents = file.read()
+          contents_seq = _dafny.Seq(contents)
+          return (False, contents_seq, _dafny.Seq())
+    except:
+      exc_str = traceback.format_exc()
+      exc_seq = _dafny.Seq(exc_str)
+      return (True, _dafny.Seq(), exc_seq)
+  
+  @staticmethod
+  def INTERNAL__WriteBytesToFile(path, contents):
+    path_str = path.VerbatimString(False)
+    contents_bytes = bytes(contents)
+
+    try:
+      os.makedirs(os.path.basename(path_str), exist_ok=True)
```

Comment:
It's preferable to use [`pathlib.Path(...).mkdir(parents=True, exist_ok=True)`](https://docs.python.org/3/library/os.html#os.makedirs) as it won't get tripped up if `..` (or the OS equivalent) appears in the path (see the ["Note" in os.makedirs](https://docs.python.org/3/library/os.html#os.makedirs)). Alternatively, you could [normalize/absolutize](https://docs.python.org/3/library/os.html#os.makedirs) the path before passing to `os.makedirs`.

Reply:
Good catch, thanks (added `..` to the test cases as well)